Sources/Impl/PinLayoutImpl+Coordinates.swift

```diff
@@ -24,6 +24,19 @@ extension PinLayoutImpl {
     internal func top(_ context: Context) {
         setTop(0, context)
     }
+
+    @discardableResult
+    internal func top(_ value: CGFloat, _ context: Context) -> PinLayout {
+        setTop(value, context)
+        return self
+    }
+
+    @discardableResult
+    internal func top(_ percent: Percent, _ context: Context) -> PinLayout {
+        guard let layoutSuperviewRect = layoutSuperviewRect(context) else { return self }
```

Comment:
Could you update the method `func top(_ percent: Percent)-> PinLayout` in PinLayoutImpl.swift to:
```
func top(_ percent: Percent) -> PinLayout {
        return top(percent,  { return "top(\(percent.description))" })
    }
```